modules/inventory/server.lua

```diff
@@ -1342,6 +1342,37 @@ function Inventory.RemoveItem(inv, item, count, metadata, slot, ignoreTotal)
 end
 exports('RemoveItem', Inventory.RemoveItem)
 
+---@param inv inventory
+---@param items table
```

Comment:
`---@param items table<string, number>` would be more accurate, however it would make more sense for both count _and_ metadata to be tied to the item. The current structure also wouldn't support adding two items with the same name, but separate metadata.

This would be better.

```lua
exports.ox_inventory:CanCarryItems(invId, {
   { 'bread', 1, { durability = 0 } },
   { 'bread', 1, { durability = 100 } },
})
```

Reply:
Why do you want metadata to check if a player can carry an item, I think simply just checking if they have enough weight and available slots should be enough.

Reply:
1. Items with different metadata cannot stack with eachother.
2. Metadata can adjust item weight (consider a water bottle that empties overtime).